tools/pagerduty-migrator/migrator/resources/integrations.py

```diff
@@ -52,7 +55,14 @@ def migrate_integration(integration: dict, escalation_policies: list[dict]) -> N
 def create_integration(
     name: str, integration_type: str, escalation_chain_id: str
 ) -> None:
-    payload = {"name": name, "type": integration_type, "team_id": None}
+
+    DEFAULT_INTEGRATION_TYPE = list(PAGERDUTY_TO_ONCALL_VENDOR_MAP.values())
+    
+    payload = {
+        "name": name,
+        "type": integration_type if integration_type in DEFAULT_INTEGRATION_TYPE else FALLBACK_INTEGRATION_TYPE,
```

Comment:
I think this introduces the 400 error, so removing this as per previous comment should resolve the issue.